.github/workflows/ci/pkg/bot/check.go

```diff
@@ -237,13 +237,13 @@ func dismissMessage(pr *environment.Metadata, required []string) string {
 	var sb strings.Builder
 	sb.WriteString("new commit pushed, please re-review ")
 	for _, reviewer := range required {
-		sb.WriteString(fmt.Sprintf("@%s", reviewer))
+		sb.WriteString(fmt.Sprintf("@%s ", reviewer))
 	}
 	return sb.String()
```

Comment:
If you don't want the final space after the last reviewer:

```suggestion
	return strings.TrimSpace(sb.String())
```